Lib/test/test_idle/__init__.py

```diff
@@ -1,8 +1,10 @@
 import unittest
+import os
+
+from test import support
```

Comment:
This and the next change are pure style preference, independent of this issue.  I don't care either way.